cobc/cconv.c

```diff
@@ -223,3 +262,19 @@ const cob_u8_t cob_ascii_alt_ebcdic[256] = {
 	0xF0, 0xF1, 0xF2, 0xF3, 0xF4, 0xF5, 0xF6, 0xF7,
 	0xF8, 0xF9, 0xFA, 0xFB, 0xFC, 0xFD, 0xFE, 0xFF
 };
+
+/* Decipher character conversion table names */
+int cobc_deciph_ebcdic_table_name (enum cb_ebcdic_table * const t,
```

Comment:
I suggest to return the enum value (and a new `CB_EBCDIC_UNDEFINED` in the error case) here.

Reply:
Having an explicit undefined in the enumeration goes all the way to `codegen`, where specific error cases are then required on each test of a conversion table. Is this really what we want?

Reply:
```c
		case 14: {
			/* -febcdic-table=<cconv-table> */
			const enum ebcdic_table cb_ebcdic_table = cobc_deciph_ebcdic_table_name (cob_optarg);
			if (ebcdic_table == SOME_UNDEFINED) {
				cobc_err_exit (COBC_INV_PAR, "-febcdic-table");
			}
			break;
		}
```

:-)

Reply:
I mean, every check on `cb_ebcdic_table` in `codegen` need to deal with `UNDEFINED` if that is part of the enumeration type, not just the code that checks the option argument.

Reply:
Anyways, with `cb_ebcdic_table` encapsulated in `cconv` that aspect is moot ;-)

Reply:
Sure the C way of handling enumerations is quite poor. But I usually operate under the assumption that, in a program where the only possible assignments to an enum variable satisfy some strict discipline (like only using well-declared named constants), then uses of that variable may only need to consider declared cases.

However, that's considerations from a dev that is more used to much higher-level (functional) languages ;-)